packages/messaging/lib/webkit.js

```diff
@@ -59,11 +59,6 @@ import { isResponseFor, isSubscriptionEventFor } from '../schema.js'
  * @implements {MessagingTransport}
  */
 export class WebkitMessagingTransport {
-    /** @type {WebkitMessagingConfig} */
-    config
```

Comment:
Do we need these changes still?

Reply:
These were always assigned in the constructor, so currently served no purpose.

Reply:
I guess, just seems unrelated but it's fine.

Reply:
it's not unrelated - these were the first errors I encountered (until I dug deeper), so when evaluating them and realising they were not needed, made sense to remove as part of this work :)